lighthouse-cli/test/fixtures/perf/unsized-images.html

```diff
@@ -0,0 +1,19 @@
+<html>
```

Comment:
license? we're pretty inconsistent about it non-JS files, so you might need to hunt a smidge before finding one